posthog/migrations/0127_stricter_team.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.0.6 on 2021-02-09 09:11
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+def prepare_teams(apps, schema_editor):
+    Team = apps.get_model("posthog", "Team")
+    Organization = apps.get_model("posthog", "Organization")
+    first_organization = Organization.objects.order_by("id").first()
+    if first_organization is not None:
+        Team.objects.filter(organization_id__isnull=True).update(organization_id=first_organization.id)
+    else:
+        Team.objects.filter(organization_id__isnull=True).delete()
```

Comment:
Is there any chance this will delete someone's entire database? 🤔

Reply:
Not really. Definitely not anything usable.